agent/consul/rtt.go

```diff
@@ -22,7 +22,7 @@ func (s *Server) newNodeSorter(cs lib.CoordinateSet, nodes structs.Nodes) (sort.
 	state := s.fsm.State()
 	vec := make([]float64, len(nodes))
 	for i, node := range nodes {
-		other, err := state.Coordinate(node.Node)
+		_, other, err := state.Coordinate(node.Node, nil)
```

Comment:
Why doesn't this panic since `Coordinate(` doesn't handle `ws == nil`?

Reply:
memdb.WatchSet itself has the nil checks